packages/core/src/interactions/packet/acknowledgement.ts

```diff
@@ -19,31 +19,22 @@ export function subscribeToAcknowledgements(
   async function handleAcknowledgement(msg: Uint8Array, remotePeer: PeerId) {
     const ackMsg = Acknowledgement.deserialize(msg, pubKey, remotePeer)
 
-    let unacknowledgedTicket: UnacknowledgedTicket | undefined
     try {
-      unacknowledgedTicket = await db.getUnacknowledgedTicket(ackMsg.ackChallenge)
-    } catch (err) {
-      if (!err.notFound) {
-        throw err
-      }
-    }
-
-    if (unacknowledgedTicket != undefined) {
+      let unacknowledgedTicket = await db.getUnacknowledgedTicket(ackMsg.ackChallenge)
       const channel = chain.getChannel(new PublicKey(pubKey.pubKey.marshal()), unacknowledgedTicket.signer)
-
       const ackedTicket = await channel.acknowledge(unacknowledgedTicket, ackMsg.ackKeyShare)
-
-      if (ackedTicket === null) {
-        log(`Got a ticket that is not a win. Dropping ticket.`)
-        await db.delAcknowledgedTicket(ackedTicket.ticket.challenge)
-      } else {
+      if (ackedTicket) {
         log(`Storing winning ticket`)
         await db.replaceUnAckWithAck(ackMsg.ackChallenge, ackedTicket)
       }
+    } catch (err) {
```

Comment:
The catch block also catches errors from `replaceUnAckWithAck`. Catching `getUnacknowledgedTicket` and `replaceUnAckWithAck` in seperate try/catch blocks might be better

Reply:
Why? We don't try and recover anyway. IMHO we should get rid of this try/catch.